### DISCUSSION TITLE
External MariaDB database

### DISCUSSION THREAD

**Discussion by kristianf97** [NginxProxyManager/nginx-proxy-manager#3297] [Q&A]
Hello, is there a way to setup NginxProxyManager with an external MariaDB server?

**Comment by maurxce** [comment]
https://nginxproxymanager.com/setup/#using-mysql-mariadb-database

I believe you should just be able to enter a domain/ip for the `DB_MYSQL_HOST` key

**Reply by bckelley** [reply]
> https://nginxproxymanager.com/setup/#using-mysql-mariadb-database
> 
> I believe you should just be able to enter a domain/ip for the `DB_MYSQL_HOST` key

doesn't seem to be that simple

**Reply by maurxce** [reply]
are you able to connect to the mysql host via the cli or mysql workbench (or similar)?
